app/helpers/application_helper.rb

```diff
@@ -1,2 +1,8 @@
 module ApplicationHelper
+  def flash_class(level)
+    case level.to_sym
+      when :notice then 'success'
+      when :alert then 'danger'
+    end
+  end
```

Comment:
I know, trying to find out how that is done, unless someone else beats me to it..